share/doc/wake/logging.md

```diff
@@ -0,0 +1,65 @@
+# Logging
+
+This article gives a high level overview of how wake logging works.
+Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
+for commented code.
+
+## How Logging works on Wake Execution commands
+
+Every `Job` will emit a stderr and a stdout.
```

Comment:
I think "a" vs "an" depends how you say this. If you say "stood error" then "a" is correct but if you're like me and say "es-tee-dee error" then you want "an" here.

Maybe to be more widely clear if you said "ever `Job` will emit some (possibly empty) string to its stderr and stdout" it might be more explict and avoid the pronunciation ambiguity.